### ISSUE TITLE
Is it possible to replace spaces in the search query with "%20" instead of "+"?

### ISSUE THREAD

**Issue by Xebtria** [evilpie/add-custom-search-engine#63]
I am in a dilemma here. I added two custom search engines for dict.cc and deepl.com (makes it easier to translate stuff).

if I do that with dict.cc it works fine:
![image](https://github.com/evilpie/add-custom-search-engine/assets/8898076/c24bac61-5909-42db-9010-209ed1067810)
![image](https://github.com/evilpie/add-custom-search-engine/assets/8898076/a9e25a68-d528-49ec-96e6-7ecf2cf82dfb)
![image](https://github.com/evilpie/add-custom-search-engine/assets/8898076/f7e8d186-750d-416d-a6e9-fb763ac326ea)
as you can see, it changes the spaces to "+", and these work with dict.cc. as a matter of fact, if I manually search multi word terms on dict.cc it does replace the spaces with + by itself.

however, deepl works a bit different - if you do a manual search with deepl it does not add "+" signs, it just leaves it as spaces:
![image](https://github.com/evilpie/add-custom-search-engine/assets/8898076/a08c73de-e87f-4c32-8483-d93d499f1d65)

but if I do that custom search with deepl, it adds "+" signs as spaces:
![image](https://github.com/evilpie/add-custom-search-engine/assets/8898076/2897dd29-7963-40d9-a561-6d6c8099e3a1)
![image](https://github.com/evilpie/add-custom-search-engine/assets/8898076/fd43ae42-ffc0-4691-b5a2-80cf3bdea06f)
which, as you can see, results in the translation no longer working

If I basically manually type the unicode character for a space - %20 - in the search query
![image](https://github.com/evilpie/add-custom-search-engine/assets/8898076/5723317e-89c3-471e-84e5-2947ee64d528)
it works again:
![image](https://github.com/evilpie/add-custom-search-engine/assets/8898076/4ca712d5-2bef-4771-8a6a-47940faec124)

and if I would do the same for dict.cc:
![image](https://github.com/evilpie/add-custom-search-engine/assets/8898076/13aa0909-f6f5-4a04-99ec-931eb3a33167)
it even works here:
![image](https://github.com/evilpie/add-custom-search-engine/assets/8898076/b90c6f62-03cc-489f-82ea-6b537ff68dc8)

**long story short:** would it be possible to change the search query variable "%s" here
![image](https://github.com/evilpie/add-custom-search-engine/assets/8898076/b14248d0-6984-4b2e-9796-30da6397aa4d)
to replace entered spaces as "%20" instead of "+", or is this a firefox thing that would need to be changed by mozilla?

**Comment by evilpie** [comment]
Duplicate of #51.
